composer.json

```diff
@@ -1,5 +1,10 @@
 {
-  "require": {
-    "squizlabs/php_codesniffer": "*"
-  }
+    "require": {
```

Comment:
Remove the normal require section. That is for packages required for you app to run.